File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processactions/ProcessActionModel.ts
import { Process } from '..';
import { ProcessData } from '../processes';
import { MultilingualString, GuidValue } from '@omnia/fx-models';


export interface ProcessActionModel {
    process: Process;
    processData: { [processStepId: string]: ProcessData };
    deletedProcessId?: GuidValue;
}

File: src/Omnia.ProcessManagement.Models/ProcessActions/ProcessActionModel.cs
using Omnia.Fx.Models.Language;
using Omnia.ProcessManagement.Models.Processes;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.ProcessActions
{
    public class ProcessActionModel
    {
        public Process Process { get; set; }
        public Dictionary<Guid, ProcessData> ProcessData { get; set; }
        public Guid? DeletedProcessId { get; set; }
    }
}
